### DISCUSSION TITLE
I'm not sure how to write a test for navigating to the edit page when using `EditAction::make()`.

### DISCUSSION THREAD

**Discussion by bzy107** [filamentphp/filament#9300] [Help] [answered]
I have a table where the icon to navigate to the edit page is displayed using `EditAction::make()`.

```php
public static function table(Table $table): Table
{
    return $table
        ...
        ->actions([
            Tables\Actions\EditAction::make(),
        ])
```

In this case, only the link to the edit page is displayed.

I want to write a test for this situation, but the documentation seems to provide tests for modal displays using EditAction::make()->action().
[Here's the link to the documentation.](https://filamentphp.com/docs/3.x/tables/testing#calling-actions)

How can I write a test to confirm navigation to the edit page?

**Comment by sinnbeck** [comment, marked as answer]
Its just a regular link so there isnt really any way to test clicking it without needing some sort of E2E test. But you can check if the link is at least shown in the table. Something like
```php
it('can see edit link post', function () {
    $posts = Post::factory(2)->create();

    livewire(PostResource\Pages\ListPosts::class)
        ->assertSee(PostResource::getUrl('edit', [
            'record' => $post->first(),
        ]));
});

**Reply by bzy107** [reply]
Thank you. 
I understand that it's just a matter of displaying the link. 
I will close this discussion.
